.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "main", "master" ]
+  schedule:
+    - cron:  '*/15 * * * *'
+  workflow_dispatch:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          ref: "hp-pages"
+          path: "pages"
+      - uses: actions/checkout@v3
+        with:
+          repository: "SerenityOS/serenity"
+          path: "serenity"
+          fetch-depth: "0"
```

Comment:
fetch-depth 0 doesn't make too much sense. Shouldn't this break printing the full commit log? Also, when checking out older commits later on, the old commits aren't available, thus breaking everything.

Reply:
according to https://github.com/actions/checkout: `Set fetch-depth: 0 to fetch all history for all branches and tags`. I suppose you interpreted as no history at all ?

Reply:
Oh! Yeah, I absolutely misunderstood it then. My bad!